### ISSUE TITLE
docz:dev target fails

### ISSUE THREAD

**Issue by ggascoigne** [JohannesKlauss/react-hotkeys-hook#487]
To be honest this isn't likely to be your bug, but as I know nothing about gatsby I'm thinking that you'll have a better idea if there's an easy way to fix things.

This was a fresh clone, followed by a `yarn install` then `yarn docz:dev`

```
❯ yarn docz:dev
yarn run v1.22.5
$ gatsby develop
success open and validate gatsby-configs - 0.918s
warn Plugin gatsby-source-filesystem is not compatible with your gatsby version 3.0.2 - It requires gatsby@^2.2.0
warn Plugin gatsby-plugin-react-helmet-async is not compatible with your gatsby version 3.0.2 - It requires gatsby@2.x
warn Plugin gatsby-plugin-emotion is not compatible with your gatsby version 3.0.2 - It requires gatsby@^2.0.0
success load plugins - 1.213s
success onPreInit - 0.035s
success initialize cache - 0.025s
success copy gatsby files - 0.166s
success onPreBootstrap - 0.351s
success createSchemaCustomization - 0.028s
success Checking for changed pages - 0.001s
success source and transform nodes - 1.670s

 ERROR

Missing onError handler for invocation 'building-schema', error was 'Error: TypeError[File.publicURL]: Cannot convert to OutputType the following value: Object({ type: String, args: Object({  }),
description: "Copy file to static directory and return public url to it", resolve: [function resolve] })'. Stacktrace was 'Error: TypeError[File.publicURL]: Cannot convert to OutputType the
following value: Object({ type: String, args: Object({  }), description: "Copy file to static directory and return public url to it", resolve: [function resolve] })
    at TypeMapper.convertOutputFieldConfig (/Users/ggp/dev/git/react-hotkeys-hook/node_modules/gatsby/node_modules/graphql-compose/lib/TypeMapper.js:320:13)
    at ObjectTypeComposer.setField (/Users/ggp/dev/git/react-hotkeys-hook/node_modules/gatsby/node_modules/graphql-compose/lib/ObjectTypeComposer.js:215:114)
    at /Users/ggp/dev/git/react-hotkeys-hook/node_modules/gatsby/node_modules/graphql-compose/lib/ObjectTypeComposer.js:242:14
    at Array.forEach (<anonymous>)
    at ObjectTypeComposer.addNestedFields (/Users/ggp/dev/git/react-hotkeys-hook/node_modules/gatsby/node_modules/graphql-compose/lib/ObjectTypeComposer.js:235:28)
    at forEach (/Users/ggp/dev/git/react-hotkeys-hook/node_modules/gatsby/src/schema/schema.js:758:39)
    at Array.forEach (<anonymous>)
    at /Users/ggp/dev/git/react-hotkeys-hook/node_modules/gatsby/src/schema/schema.js:758:18
    at async Promise.all (index 31)
    at updateSchemaComposer (/Users/ggp/dev/git/react-hotkeys-hook/node_modules/gatsby/src/schema/schema.js:190:3)
    at buildSchema (/Users/ggp/dev/git/react-hotkeys-hook/node_modules/gatsby/src/schema/schema.js:65:3)
    at build (/Users/ggp/dev/git/react-hotkeys-hook/node_modules/gatsby/src/schema/index.js:105:18)
    at buildSchema (/Users/ggp/dev/git/react-hotkeys-hook/node_modules/gatsby/src/services/build-schema.ts:19:3)'

⠧ building schema

```

**Comment by JohannesKlauss** [comment]
Oh, thx for raising the issue. I haven't built the docs in a while. I'll check it out.

**Comment by JohannesKlauss** [comment]
Fixed with 3.2.0
